test/api/unit/models/build.test.js

```diff
@@ -52,54 +53,74 @@ describe('Build model', () => {
   });
 
   describe('after create hook', () => {
-    it('should send a build new build message', (done) => {
-      factory.build()
-        .then(build => build.updateJobStatus({ status: 'complete' }))
-        .then((build) => {
-          const queuedBuild = sendMessageStub.getCall(0).args[0];
-          const buildCount = sendMessageStub.getCall(0).args[1];
+    it('should send a build new build message', async () => {
+      const startedAt = new Date();
+      let build = await factory.build();
+      // create delay while s3 infra create will be removed with 1 bucket federalist
+      await factory.site();
```

Comment:
If the purpose of `await factory.site();` is to create the delay mentioned in the comment above, maybe we can replace this with something more semantic utility method like
```
await wait();
```
where
```
function wait(ms = 100) {
  return new Promise(resolve => setTimeout(resolve, ms));
}
```